proposal.rst

```diff
@@ -200,27 +195,27 @@ Project Details
 
 **Describe any existing work and concepts on which your project is based.**
 
-FILLME
+	I have studied with Golang in my internship. I worked on developing a software that makes web crawling.	I used PostgreSQL and Beego in this project.
+
 
 **Do you have any preliminary findings or results which suggest that your
 approach is possible and likely to succeed?**
 
-FILLME
+-
 
 **What other approaches have you considered, and why did you reject those in
 favour of your chosen approach?**
 
-FILLME
+	I thought of finding only the best fit for the approaches.
 
-**Please note any uncertainties or aspects which depend on further research or
-investigation.**
+**Please note any uncertainties or aspects which depend on further research or investigation.**
 
 FILLME
 
 **How useful will your results be when not everything works out exactly as
 planned?**
 
-FILLME
+	I think my results will be useful because I don't lose motivation in a stress situation.
```

Comment:
There's no detail at all here of what you're actually planning to do, nor how you intend to go about it.

The "project ideas" from the list are just ideas, which you need to research and flesh out into a proposal.  You need to do a lot more to make a viable proposal than just point at an idea from the list and add some brief information about yourself.